gulpfile.js

```diff
@@ -39,32 +39,48 @@ gulp.task('npminstall', () =>
     .pipe(es.mapSync(file => npmInstallTask(file.path)))
 );
 
-gulp.task('scanner:download', () =>
-  download(scanner.url)
-    .pipe(decompress())
-    .pipe(gulp.dest(paths.build.scanner))
+gulp.task('scanner:download', () => {
+    download(scanner.classicUrl)
+      .pipe(decompress())
+      .pipe(gulp.dest(paths.build.classicScanner))
+
+    download(scanner.dotnetUrl)
+      .pipe(decompress())
+      .pipe(gulp.dest(paths.build.dotnetScanner))
+  }
 );
 
 gulp.task('scanner:copy', ['scanner:download'], () => {
   const scannerFolders = [
-    path.join(paths.build.extensions.sonarqubeTasks, 'prepare', 'old', 'SonarQubeScannerMsBuild'),
-    path.join(paths.build.extensions.sonarqubeTasks, 'prepare', 'new', 'sonar-scanner-msbuild'),
-    path.join(paths.build.extensions.sonarcloudTasks, 'prepare', 'new', 'sonar-scanner-msbuild')
+    path.join(paths.build.extensions.sonarqubeTasks,  'prepare', 'old', 'SonarQubeScannerMsBuild'),
+    path.join(paths.build.extensions.sonarqubeTasks,  'prepare', 'new', 'classic-sonar-scanner-msbuild'),
+    path.join(paths.build.extensions.sonarcloudTasks, 'prepare', 'new', 'classic-sonar-scanner-msbuild'),
+  ];
+
+  const dotnetScannerFolders = [
+    path.join(paths.build.extensions.sonarqubeTasks,  'prepare', 'new', 'dotnet-sonar-scanner-msbuild'),   
+    path.join(paths.build.extensions.sonarcloudTasks, 'prepare', 'new', 'dotnet-sonar-scanner-msbuild')    
   ];
+
   const cliFolders = [
-    path.join(paths.build.extensions.sonarqubeTasks, 'scanner-cli', 'old', 'sonar-scanner'),
-    path.join(paths.build.extensions.sonarqubeTasks, 'analyze', 'new', 'sonar-scanner'),
-    path.join(paths.build.extensions.sonarcloudTasks, 'analyze', 'new', 'sonar-scanner')
+    path.join(paths.build.extensions.sonarqubeTasks,  'scanner-cli', 'old', 'sonar-scanner'),
+    path.join(paths.build.extensions.sonarcloudTasks, 'analyze',     'new', 'sonar-scanner')
```

Comment:
The sonarqube vsix contains scanner-cli in folder `dotnet-sonar-scanner`. Debugging to see where it comes from.
Otherwise, the msbuild/cli scanner files are where I'd expect them to be.